### PR TITLE
Update monitoring docs

### PR DESCRIPTION
**Pull request by Snirx1**

### CODE REVIEW

File: docs/admin/runai-setup/maintenance/alert-monitoring.md [hunk 1]
```diff
@@ -53,93 +41,82 @@ This documentation outlines the steps required to set up Alertmanager within the
...
            alertmanagers:
               - namespace: runai
-                name: alertmanager-runai
+                name: alertmanager-operated
                 port: web
 
 3. Save and exit the editor. The configuration will be automatically reloaded.
 
 ## Configuring Alertmanager for Custom Email Alerts
+In each step copy the content to a file and apply it to the cluster using `kubectl apply -f`
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/maintenance/alert-monitoring.md:70]:
```suggestion
In each step, copy the contents to a file and apply it to the cluster using `kubectl apply -f`
```
What contents? Is there a file with contents? Not sure what you want to say here. This is the start of a section and so we need to be specific about the user needs to copy.

File: docs/admin/runai-setup/maintenance/alert-monitoring.md [hunk 2]
```diff
@@ -53,93 +41,82 @@ This documentation outlines the steps required to set up Alertmanager within the
...
-3. Add to the `spec` section, a new receiver configuration to send alerts via email:
-
-        receivers:
-        - name: 'email'
-           emailConfigs:
-           - to: 'your_email@example.com'
-              from: 'alertmanager@example.com'
-              smarthost: 'smtp.yourmailprovider.com:587'
-              authUsername: 'your_username'
+2. Apply the `alertmanagerconfig`, replace the relevant smtp details with your own:
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/maintenance/alert-monitoring.md:82]:
```suggestion
2. Apply the `alertmanagerconfig`, replace the relevant smtp details with your own:
```
How do you apply? Is there a command for this?

File: docs/admin/runai-setup/maintenance/alert-monitoring.md (hunk 1 above)
Comment (Snirx1) [reply docs/admin/runai-setup/maintenance/alert-monitoring.md:70]: I mean the things in the code snippets, not sure how to refer to them, what do you think?

File: docs/admin/runai-setup/maintenance/alert-monitoring.md [hunk 3]
```diff
@@ -53,93 +41,82 @@ This documentation outlines the steps required to set up Alertmanager within the
...
-              nodePort: 30903
-              port: 9093
-              protocol: TCP
-              targetPort: web
-           selector:
-              alertmanager: runai
-        EOF
+        kubectl get alertmanager -n runai
+
+3. Validate that prometheus operator has created a service for alertmanager:
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/maintenance/alert-monitoring.md:49]:
```suggestion
3. Validate that the prometheus operator has created a service for alertmanager:
```

File: docs/admin/runai-setup/maintenance/alert-monitoring.md [hunk 4]
```diff
@@ -53,93 +41,82 @@ This documentation outlines the steps required to set up Alertmanager within the
...
            alertmanagers:
               - namespace: runai
-                name: alertmanager-runai
+                name: alertmanager-operated
                 port: web
 
 3. Save and exit the editor. The configuration will be automatically reloaded.
 
 ## Configuring Alertmanager for Custom Email Alerts
+In each step, copy the contents to a file and apply it to the cluster using `kubectl apply -f`
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/maintenance/alert-monitoring.md:70]: If you are referring to code snippiets, then you need to specify that, and more specifically which ones need to be copied, to which file, and to where in the file. If this a new file then that needs to be stated, if it is an already existing file, then you need to be specific about the name of the file.

File: docs/admin/runai-setup/maintenance/alert-monitoring.md [hunk 5]
```diff
@@ -53,93 +41,82 @@ This documentation outlines the steps required to set up Alertmanager within the
...
+           name: alertmanager-smtp-password
            namespace: runai
         stringData:
            password: "your_smtp_password"
-        EOF
-
-2. Edit the Alertmanager configuration:
 
-       `kubectl edit alertmanagerconfig -n runai`
+2. Apply the `alertmanagerconfig`, replace the relevant smtp details with your own (check for indentation problems before applying):
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/maintenance/alert-monitoring.md:82]: This is backwards:
```suggestion
2.   Replace the relevant smtp details with your own, then apply the `alertmanagerconfig`. Check for indentaion issues before applying.
```

File: docs/admin/runai-setup/maintenance/alert-monitoring.md [hunk 6]
```diff
@@ -53,93 +41,82 @@ This documentation outlines the steps required to set up Alertmanager within the
...
-           
-           matchers:
-           - matchType: =~
-             name: alertname
-             value: Runai.*
-        
-           receiver: email
-
-2. Save and exit the editor. The configuration will be automatically reloaded.
+3. Save and exit the editor. The configuration will be automatically reloaded.
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/maintenance/alert-monitoring.md:122]:
```suggestion
3. Save and exit the editor. The configuration will automatically be reloaded.
```

**Review by jasonnovichRunAI — COMMENTED** [review]
Grammar issues.

File: docs/admin/runai-setup/maintenance/alert-monitoring.md (hunk 2 above)
Comment (Snirx1) [reply docs/admin/runai-setup/maintenance/alert-monitoring.md]: added a description

File: docs/admin/runai-setup/maintenance/alert-monitoring.md (hunk 4 above)
Comment (Snirx1) [reply docs/admin/runai-setup/maintenance/alert-monitoring.md]: added

File: docs/admin/runai-setup/maintenance/alert-monitoring.md [hunk 7]
```diff
@@ -27,22 +27,10 @@ This documentation outlines the steps required to set up Alertmanager within the
        You should see the Prometheus instance(s) listed along with their status.
 
 ## Enabling Alertmanager
+In each step, copy the contents of the code snippets to a new file and apply it to the cluster using `kubectl apply -f` command
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/maintenance/alert-monitoring.md:30]:
```suggestion
In each of the steps in this section, copy the contents of the code snippets to a new file and apply it to the cluster using `kubectl apply -f`.
```

File: docs/admin/runai-setup/maintenance/alert-monitoring.md [hunk 8]
```diff
@@ -53,93 +41,82 @@ This documentation outlines the steps required to set up Alertmanager within the
...
            alertmanagers:
               - namespace: runai
-                name: alertmanager-runai
+                name: alertmanager-operated
                 port: web
 
 3. Save and exit the editor. The configuration will be automatically reloaded.
 
 ## Configuring Alertmanager for Custom Email Alerts
+In each step, copy the contents of the code snippets to a new file and apply it to the cluster using `kubectl apply -f` command
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/maintenance/alert-monitoring.md:70]:
```suggestion
In each step, copy the contents of the code snippets to a new file and apply it to the cluster using `kubectl apply -f`.
```

File: docs/admin/runai-setup/maintenance/alert-monitoring.md [hunk 9]
```diff
@@ -53,93 +41,82 @@ This documentation outlines the steps required to set up Alertmanager within the
...
+           name: alertmanager-smtp-password
            namespace: runai
         stringData:
            password: "your_smtp_password"
-        EOF
-
-2. Edit the Alertmanager configuration:
 
-       `kubectl edit alertmanagerconfig -n runai`
+2.   Replace the relevant smtp details with your own, then apply the `alertmanagerconfig` using `kubectl apply` command. Check for indentation issues before applying.
```
Comment (jasonnovichRunAI) [inline docs/admin/runai-setup/maintenance/alert-monitoring.md:82]:
```suggestion
2.   Replace the relevant smtp details with your own, then apply the `alertmanagerconfig` using `kubectl apply`. Check for indentation issues before applying.
```

**Pull request merged**